
Showing logs for build [32mmeta-salaboy-test-helm-pipeline-zsvpv-1[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-44sz4[0m
{"level":"info","ts":1591695890.3147235,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1591695890.3150904,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-salaboy-test-helm-pipeline-zsvpv-1[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-jwhgr[0m
{"level":"info","ts":1591695891.3441114,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1591695891.3461607,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-salaboy-test-helm-pipeline-zsvpv-1[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-salaboy-test-helm-pipeline-zsvpv-1[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-salaboy-test-helm-pipeline-zsvpv-zz7hv[0m
{"level":"info","ts":1591695896.4666994,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"error","ts":1591695896.856702,"logger":"fallback-logger","caller":"git/git.go:40","msg":"Error running git [fetch --depth=1 --recurse-submodules=yes origin pr-1]: exit status 128\nfatal: couldn't find remote ref pr-1\n","stacktrace":"github.com/tektoncd/pipeline/pkg/git.run\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/pkg/git/git.go:40\ngithub.com/tektoncd/pipeline/pkg/git.Fetch\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/pkg/git/git.go:91\nmain.main\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/cmd/git-init/main.go:39\nruntime.main\n\t/usr/local/Cellar/go@1.12/1.12.17/libexec/src/runtime/proc.go:200"}
{"level":"error","ts":1591695897.6664193,"logger":"fallback-logger","caller":"git/git.go:40","msg":"Error running git [pull --recurse-submodules=yes origin]: exit status 1\nFrom https://github.com/*******/test-helm-pipeline\n * [new branch]      gh-pages   -> origin/gh-pages\n * [new branch]      master     -> origin/master\n * [new branch]      test-pr    -> origin/test-pr\nYou asked to pull from the remote 'origin', but did not specify\na branch. Because this is not the default configured remote\nfor your current branch, you must specify a branch on the command line.\n","stacktrace":"github.com/tektoncd/pipeline/pkg/git.run\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/pkg/git/git.go:40\ngithub.com/tektoncd/pipeline/pkg/git.Fetch\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/pkg/git/git.go:94\nmain.main\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/cmd/git-init/main.go:39\nruntime.main\n\t/usr/local/Cellar/go@1.12/1.12.17/libexec/src/runtime/proc.go:200"}
{"level":"warn","ts":1591695897.6667643,"logger":"fallback-logger","caller":"git/git.go:95","msg":"Failed to pull origin : exit status 1"}
{"level":"error","ts":1591695897.672547,"logger":"fallback-logger","caller":"git/git.go:40","msg":"Error running git [checkout pr-1]: exit status 1\nerror: pathspec 'pr-1' did not match any file(s) known to git\n","stacktrace":"github.com/tektoncd/pipeline/pkg/git.run\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/pkg/git/git.go:40\ngithub.com/tektoncd/pipeline/pkg/git.Fetch\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/pkg/git/git.go:97\nmain.main\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/cmd/git-init/main.go:39\nruntime.main\n\t/usr/local/Cellar/go@1.12/1.12.17/libexec/src/runtime/proc.go:200"}
{"level":"fatal","ts":1591695897.6725945,"logger":"fallback-logger","caller":"git-init/main.go:40","msg":"Error fetching git repository: exit status 1","stacktrace":"main.main\n\t/Users/abayer/go/src/github.com/tektoncd/pipeline/cmd/git-init/main.go:40\nruntime.main\n\t/usr/local/Cellar/go@1.12/1.12.17/libexec/src/runtime/proc.go:200"}
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-source-meta-salaboy-test-helm-pipeline-zsvpv-zz7hv'. The execution of the pipeline has stopped.[0m
